hal/armv8m/nrf/types.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * Types
+ *
+ * Copyright 2021, 2022 Phoenix Systems
+ * Author: Hubert Buczynski, Gerard Swiderski, Aleksander Kaminski, Damian Loewnau
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef _TYPES_H_
+#define _TYPES_H_
+
+
+#define NULL 0
```

Comment:
Because of change: https://github.com/phoenix-rtos/plo/commit/23930cca8d504d63bfc01619ae52951ff957d5b1
```suggestion
#define NULL ((void *)0)
```

Reply:
Thanks, applied